core/src/main/scala/org/apache/spark/scheduler/ExecutorLossReason.scala

```diff
@@ -56,7 +56,7 @@ private [spark] object LossReasonPending extends ExecutorLossReason("Pending los
  * @param workerLost whether the worker is confirmed lost too (i.e. including shuffle service)
  */
 private[spark]
-case class SlaveLost(_message: String = "Slave lost", workerLost: Boolean = false)
+case class ExecutorProcessLost(_message: String = "Worker lost", workerLost: Boolean = false)
```

Comment:
I think the default message should be "Executor Process Lost". Having it be "Worker lost" signals that workerLost flag is true. Ie, this would look confusing if the message is "Worker lost" and the workerLost = false.